src/components/draw/feature-measurements.vue

```diff
@@ -58,21 +77,32 @@ watchEffect(async () => {
   }
 })
 
-function onClickValidateRadius() {
-  alert('TODO: validate /save radius')
+watchEffect(() => {
+  inputRadius.value =
+    featureType.value === 'drawnCircle'
+      ? getCircleRadius(featureGeometry.value as Circle, mapProjection).toFixed(
```

Comment:
is this returning a number? because from the declearation above I see :  `inputRadius: Ref<string>`

Reply:
The `toFixed()` converted it into `string`, but I changed everything into a `number` as you suggested below.